### DISCUSSION TITLE
Project wide Search/Report of max-length violations

### DISCUSSION THREAD

**Discussion by rsr-cs** [WeblateOrg/weblate#7160] [Q&A] [answered]
Hi Weblate Experts,

ist there any possibility to check a translation project for violations of the maximum length of target strings? In our case this is reflected by the maxwidth-attribute in XLIFF-files. I am thinking of a global search or kind of report.

Thanks for any hints

Reiner

**Comment by nijel** [comment, marked as answer]
Tools / Failing checks will get you to detailed overview of checks.

**Reply by rsr-cs** [reply]
In our hosted environment there are no entries in Tools/Failing checks. Do we need to activate it additionally?

**Reply by nijel** [reply]
Ah. AFAIR you wanted to disable checks for performance reasons years ago...

**Reply by rsr-cs** [reply]
Oh :-) Couldn't find anything in our records. but if this affects performance of course we should be careful.

**Reply by nijel** [reply]
It indeed slows down importing strings into weblate because it's additional processing for every string. On the other side it is quite useful feature, and I'm not aware of any other installation which would have them disabled.

**Reply by rsr-cs** [reply]
Okay, sounds good. I will contact you directly via Weblate support on how to activate the feature again.

**Reply by nijel** [reply]
This is kind of view you can expect then: https://hosted.weblate.org/checks/?project=weblate
